tutor/templates/k8s/deployments.yml

```diff
@@ -155,6 +155,8 @@ spec:
               value: "{{ MONGODB_PORT }}"
             - name: MONGODB_DATABASE
               value: "{{ FORUM_MONGODB_DATABASE }}"
+            - name: MONGOID_AUTH_MECH
+              value: "{% if MONGODB_USERNAME and MONGODB_PASSWORD %}{{ MONGOID_AUTH_MECH }}{% endif %}"
```

Comment:
From my understanding, it is unnecessary to add the `{% if MONGODB_USERNAME and MONGODB_PASSWORD %}` condition. Can you confirm?